### DISCUSSION TITLE
Help with Sonoff Dual R2 to control shutters

### DISCUSSION THREAD

**Discussion by rmcbc** [arendst/Tasmota#14927] [Q&A] [answered]
### PROBLEM DESCRIPTION
Sonoff Dual R2 to control shutters

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Sonoff Dual R2
- [x] Tasmota binary firmware version number used: 11.0.0
  - [x] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: EasyESP, PlatformIO and Web page
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
22:04:28.110 MQT: tasmota/EstoreEscritorio41/RESULT = {"NAME":"Sonoff Dual R2 - Shades","GPIO":[0,1,0,32,0,225,33,0,224,320,0,0,0,0],"FLAG":0,"BASE":39}
22:04:28.332 MQT: tasmota/EstoreEscritorio41/RESULT = {"Module":{"0":"Sonoff Dual R2 - Shades"}}
22:04:28.585 MQT: tasmota/EstoreEscritorio41/RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"32":"Button1"},"GPIO4":{"0":"None"},"GPIO5":{"225":"Relay2"},"GPIO9":{"33":"Button2"},"GPIO10":{"0":"None"},"GPIO12":{"224":"Relay1"},"GPIO13":{"320":"Led_i1"},"GPIO14":{"0":"None"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}
```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
22:06:10.575 MQT: tasmota/EstoreEscritorio41/STATUS = {"Status":{"Module":0,"DeviceName":"EstoreEscritorio41","FriendlyName":["EstoreEscritorio41","SonOffBasic092"],"Topic":"EstoreEscritorio41","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
22:06:10.611 MQT: tasmota/EstoreEscritorio41/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-lite.bin.gz","RestartReason":"Software/System restart","Uptime":"0T00:08:44","StartupUTC":"2022-02-20T20:57:26","Sleep":50,"CfgHolder":4617,"BootCount":7,"BCResetTime":"2022-02-20T21:21:59","SaveCount":32,"SaveAddress":"F6000"}}
22:06:10.651 MQT: tasmota/EstoreEscritorio41/STATUS2 = {"StatusFWR":{"Version":"11.0.0(tasmota)","BuildDateTime":"2022-02-20T20:26:02","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8285","CR":"460/699"}}
22:06:10.669 MQT: tasmota/EstoreEscritorio41/STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["HomeWIFIP1","HomeWIFI"],"TelePeriod":300,"Resolution":"558180C0","SetOption":["0000A00B","2805C80001000600003C5A0A190000000000","40000081","00006000","00004000"]}}
22:06:10.704 MQT: tasmota/EstoreEscritorio41/STATUS4 = {"StatusMEM":{"ProgramSize":620,"Free":380,"Heap":24,"ProgramFlashSize":1024,"FlashSize":1024,"FlashChipId":"144051","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8FDAC787","04368001","000000CF","010013C0","C000F981","00004004","00001000","00000020"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,16,18,19,20,21,22,24,26,27,29,30,35,37,45","Sensors":"1,2,3,4,5,6"}}
22:06:10.739 MQT: tasmota/EstoreEscritorio41/STATUS5 = {"StatusNET":{"Hostname":"EstoreEscritorio41-3201","IPAddress":"192.168.1.41","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.1.1","DNSServer2":"0.0.0.0","Mac":"60:01:94:CC:CC:81","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
22:06:10.764 MQT: tasmota/EstoreEscritorio41/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.1.220","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_CCCC81","MqttUser":"mosquitto","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
22:06:10.790 MQT: tasmota/EstoreEscritorio41/STATUS7 = {"StatusTIM":{"UTC":"2022-02-20T21:06:10","Local":"2022-02-20T22:06:10","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"+01:00","Sunrise":"07:49","Sunset":"18:18"}}
22:06:10.811 MQT: tasmota/EstoreEscritorio41/STATUS10 = {"StatusSNS":{"Time":"2022-02-20T22:06:10","Shutter1":{"Position":0,"Direction":0,"Target":0,"Tilt":0}}}
22:06:10.825 MQT: tasmota/EstoreEscritorio41/STATUS11 = {"StatusSTS":{"Time":"2022-02-20T22:06:10","Uptime":"0T00:08:44","UptimeSec":524,"Heap":24,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","Wifi":{"AP":1,"SSId":"HomeWIFIP1","BSSId":"DE:47:32:25:2B:1C","Channel":8,"Mode":"11n","RSSI":100,"Signal":-48,"LinkCount":1,"Downtime":"0T00:00:03"}}}
22:06:10.858 MQT: tasmota/EstoreEscritorio41/STATUS13 = {"StatusSHT":{"SHT0":{"Relay1":1,"Relay2":2,"Open":100,"Close":100,"50perc":50,"Delay":0,"Opt":"0000","Calib":[300,500,700,900,1000],"Mode":"0"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
Hi, I have a Sonoff Dual R2 to control my shutters.
The system has two external buttons connected to GPIO3 (UP) and GPIO9 (DOWN)
Relay 1 powers up the Shutter motor to move up and relay 2 powers up the shutter to move the down.
So according to this wiring I can not have relay 1 and relay 2 turn on simultaneously.
Configuration of the template:
![image](https://user-images.githubusercontent.com/16247792/154863655-b479d73a-c9f7-4278-b27d-8d3a2bb2467d.png)
This works with the buttons with the SetOption80 0:
![image](https://user-images.githubusercontent.com/16247792/154863793-277d65f1-e4f4-4f08-867b-b6053eec8380.png)
If I use " Backlog SetOption80 1; ShutterMode0 ; ShutterButton1 1 up 1 ; ShutterButton1 2 down 1 " the first button moves the shutter up, and when I press the second button both relays turn ON.
![image](https://user-images.githubusercontent.com/16247792/154864134-37b4b477-6266-47a7-9fa0-b0c659ce8fe0.png)
Can some one help me with all the steps that I need to do the correct configuration, I try a lot of things but can't understand the correct configuration for this to work.

### EXPECTED BEHAVIOUR
Work as a shutter

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by SteWers** [comment, marked as answer]
Did you do this?

Shutter mode 1 - Normal Operation
Relay1: UP/OFF, Relay2: DOWN/OFF

**Interlock 1,2** (Interlocked relay pair)

**Interlock ON**

**Comment by rmcbc** [comment]
Thanks, I forgot "Interlock 1,2"
